### ISSUE TITLE
Cannot use placeholders in slot names

### ISSUE THREAD

**Issue by FreadFries** [zorbeytorunoglu/thebestranks#17]
So i was editing the menu and wanted to make it so you can see how you would look when you reach a certain rank, but then I realised when i put the placeholder %player_name% in the name of the item that represents the rank, it just doesnt show up.

I have PlaceHolderAPI v2.11.6 and kLib installed.

**Comment by RexSE7EN** [comment]
Do you mean on displayname?

**Comment by FreadFries** [comment]
Correct

**Comment by zorbeytorunoglu** [comment]
I don't remember if I put PAPI support in GUI or not. I'll check and respond.

**Comment by RexSE7EN** [comment]
No, it doesn't. I changed the code in fork to have it accept PAPI. ~~Now to lore and messages.~~

**Update: Lore and displayname support placeholders**
